### ISSUE TITLE
BiglyBT is crashing my inet everytime, pls help

### ISSUE THREAD

**Issue by lahabe** [BiglySoftware/BiglyBT#1600]
OS: Windows 10
BiglyBT: version 2.3.0.0

Hi, everytime since I first downloaded biglybt a long time ago, it cripples my inet. I does not matter if the speed of my downloads are high or low. Whenever BiglyBT is open on my desktop(cabled) I can barely enter basic webpages, the loading time is soo long. And my wifi will not work at all while BT is running. I have a designated port opened to BT, and it is activated in settings, but still nada. Vuze was never like this for me. And whenever I download a bigger torrent which i will get max download spead on, my entire inet crashes. I got a 500/500 line .

Any sudgestions in what may be wrong for me?

**Comment by Dyras** [comment]
Try reducing the amount of maximum connections

**Comment by Makidb** [comment]
I have more or less the same issue
If biblybt is running, i have deconnections when i play
otherwise it's fine, even if i download to the point the max bandwith is reached
Maybe the ping will be bad, but i can play
If biglybt is running i can't
So there is something with biblybt

**Comment by crogonint** [comment]
I'm not sure if this is by design here, but I am having the same issue. I've BEEN having the same issue for quite a while. I'm using a software interface that doesn't support a whole ton of connections, and just starting BiglyBT up, without any running torrents can crash the internet. Then I have to reset it, re-enable it, then wait for torrents to try to connect. Then it tries to blast every freaking tracker and peer on every freaking source it can at the same time, crashing it again. After about ten minutes of this, the wait to reconnects spread everything out far enough that it stops disconnecting long ebnough to actually connect.

This creates a really irritating dilemna where I have to seek out torrents which are NOT very popular, or everthing will just crash and nothing gets done.

I've currently been attempting to download one (1) torrent for the past three days. I recently discovered the tool under Tools . Speed Limet > View Current. This tool has shown me that BiglyBT considers my connaction as a LAN connection. So I've finally figured out how to balance my download speed.

Now my problem is that I have too many freaking connection attempts. I've killed off all of the sources and all but one tracker (which has one guy seeding the torrent). I used Process Explorer to kill off everything else that might have active connections. I've dialed down the connections in BiglyBT to ludicrous low settings. Max simultaneous outbound connection attempts: 4 and Max outstanding connections: 8

I'M STILL GETTING DISCONNECTS!

Cusing under my breath, I go out looking for a GUI for active connections, Find TCPview, download run it...

$!@#%& #*$ !*@#$&! @#(  &!@*# $*! @#$ BiglyBT still has like one-hundred and fifty-seven (157) connections!! WTH?

BiglyBT.exe	28116	TCP	James-i7	6880	James-i7	0	LISTENING										
BiglyBT.exe	28116	TCP	James-i7	26175	localhost	26176	ESTABLISHED										
BiglyBT.exe	28116	TCP	James-i7	26176	localhost	26175	ESTABLISHED										
BiglyBT.exe	28116	TCP	James-i7	26177	localhost	26178	ESTABLISHED										
BiglyBT.exe	28116	TCP	James-i7	26178	localhost	26177	ESTABLISHED										
BiglyBT.exe	28116	TCP	James-i7	26179	localhost	26180	ESTABLISHED										
BiglyBT.exe	28116	TCP	James-i7	26180	localhost	26179	ESTABLISHED										
BiglyBT.exe	28116	TCP	James-i7	26185	localhost	26186	ESTABLISHED	14	14								
BiglyBT.exe	28116	TCP	James-i7	26186	localhost	26185	ESTABLISHED										
BiglyBT.exe	28116	TCP	James-i7	26189	localhost	26190	ESTABLISHED										
BiglyBT.exe	28116	TCP	James-i7	26190	localhost	26189	ESTABLISHED										
BiglyBT.exe	28116	TCP	James-i7	26192	localhost	26193	ESTABLISHED										
BiglyBT.exe	28116	TCP	James-i7	26193	localhost	26192	ESTABLISHED										
BiglyBT.exe	28116	TCP	James-i7	26200	localhost	26201	ESTABLISHED										
BiglyBT.exe	28116	TCP	James-i7	26201	localhost	26200	ESTABLISHED										
BiglyBT.exe	28116	TCP	James-i7	26202	localhost	26203	ESTABLISHED										
BiglyBT.exe	28116	TCP	James-i7	26203	localhost	26202	ESTABLISHED										
BiglyBT.exe	28116	TCP	James-i7	27182	localhost	27183	ESTABLISHED										
BiglyBT.exe	28116	TCP	James-i7	27183	localhost	27182	ESTABLISHED										
BiglyBT.exe	28116	TCP	James-i7	27184	localhost	27185	ESTABLISHED										
BiglyBT.exe	28116	TCP	James-i7	27185	localhost	27184	ESTABLISHED										
BiglyBT.exe	28116	TCP	James-i7	27186	localhost	27187	ESTABLISHED										
BiglyBT.exe	28116	TCP	James-i7	27187	localhost	27186	ESTABLISHED										
BiglyBT.exe	28116	TCP	James-i7	29492	localhost	29493	ESTABLISHED										
BiglyBT.exe	28116	TCP	James-i7	29493	localhost	29492	ESTABLISHED										
BiglyBT.exe	28116	TCP	James-i7	29494	localhost	29495	ESTABLISHED										
BiglyBT.exe	28116	TCP	James-i7	29495	localhost	29494	ESTABLISHED										
BiglyBT.exe	28116	TCP	James-i7	29496	localhost	29497	ESTABLISHED										
BiglyBT.exe	28116	TCP	James-i7	29497	localhost	29496	ESTABLISHED										
BiglyBT.exe	28116	TCP	James-i7	29498	James-i7	0	LISTENING										
BiglyBT.exe	28116	TCP	10.xxxxx	30140	50.xxxxx	14063	ESTABLISHED										
BiglyBT.exe	28116	TCP	James-i7	39971	James-i7	0	LISTENING										
BiglyBT.exe	28116	TCP	James-i7	45100	James-i7	0	LISTENING										
BiglyBT.exe	28116	TCP	James-i7	54999	James-i7	0	LISTENING										
BiglyBT.exe	28116	TCP	James-i7	27461	James-i7	0	LISTENING										
BiglyBT.exe	28116	TCP	James-i7	41265	James-i7	0	LISTENING										
BiglyBT.exe	28116	TCP	James-i7	57394	James-i7	0	LISTENING										
BiglyBT.exe	28116	TCP	James-i7	61017	James-i7	0	LISTENING										
BiglyBT.exe	28116	UDP	James-i7	ssdp	*	*				28	9,156						
BiglyBT.exe	28116	UDP	James-i7	ssdp	*	*											
BiglyBT.exe	28116	UDP	James-i7	ssdp	*	*											
BiglyBT.exe	28116	UDP	James-i7	ssdp	*	*											
BiglyBT.exe	28116	UDP	James-i7	ssdp	*	*											
BiglyBT.exe	28116	UDP	James-i7	ssdp	*	*											
BiglyBT.exe	28116	UDP	James-i7	ssdp	*	*											
BiglyBT.exe	28116	UDP	James-i7	ssdp	*	*											
BiglyBT.exe	28116	UDP	James-i7	ssdp	*	*											
BiglyBT.exe	28116	UDP	James-i7	ssdp	*	*											
BiglyBT.exe	28116	UDP	James-i7	ssdp	*	*											
BiglyBT.exe	28116	UDP	James-i7	ssdp	*	*											
BiglyBT.exe	28116	UDP	James-i7	16680	*	*											
BiglyBT.exe	28116	UDP	James-i7	16680	*	*											
BiglyBT.exe	28116	UDP	James-i7	16680	*	*											
BiglyBT.exe	28116	UDP	James-i7	16680	*	*											
BiglyBT.exe	28116	UDP	James-i7	16680	*	*											
BiglyBT.exe	28116	UDP	James-i7	16680	*	*											
BiglyBT.exe	28116	UDP	James-i7	16680	*	*											
BiglyBT.exe	28116	UDP	James-i7	16680	*	*											
BiglyBT.exe	28116	UDP	James-i7	16680	*	*											
BiglyBT.exe	28116	UDP	James-i7	16680	*	*											
BiglyBT.exe	28116	UDP	James-i7	16680	*	*											
BiglyBT.exe	28116	UDP	james-i7.abyss	49001	*	*		1	69								
BiglyBT.exe	28116	UDP	James-i7	52111	*	*											
BiglyBT.exe	28116	UDP	James-i7	52111	*	*											
BiglyBT.exe	28116	UDP	James-i7	52111	*	*											
BiglyBT.exe	28116	UDP	James-i7	52111	*	*											
BiglyBT.exe	28116	UDP	James-i7	61017	*	*											
BiglyBT.exe	28116	UDP	James-i7	61017	*	*											
BiglyBT.exe	28116	UDP	James-i7	61017	*	*											
BiglyBT.exe	28116	UDP	James-i7	61017	*	*											
BiglyBT.exe	28116	UDP	James-i7	27461	*	*											
BiglyBT.exe	28116	UDP	James-i7	41265	*	*											
BiglyBT.exe	28116	UDP	James-i7	52111	*	*											
BiglyBT.exe	28116	UDP	James-i7	52111	*	*											
BiglyBT.exe	28116	UDP	James-i7	52111	*	*											
BiglyBT.exe	28116	UDP	James-i7	59838	*	*											
BiglyBT.exe	28116	UDP	James-i7	61017	*	*											
BiglyBT.exe	28116	UDP	James-i7	61017	*	*											
BiglyBT.exe	28116	UDP	James-i7	61017	*	*											
BiglyBT.exe	28116	TCPV6	[0:0:0:0:0:0:0:0]	27461	[0:0:0:0:0:0:0:0]	0	LISTENING										
BiglyBT.exe	28116	TCPV6	[0:0:0:0:0:0:0:0]	41265	[0:0:0:0:0:0:0:0]	0	LISTENING										
BiglyBT.exe	28116	TCPV6	[0:0:0:0:0:0:0:0]	57394	[0:0:0:0:0:0:0:0]	0	LISTENING										
BiglyBT.exe	28116	TCPV6	[0:0:0:0:0:0:0:0]	61017	[0:0:0:0:0:0:0:0]	0	LISTENING										
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	1900	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	1900	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	1900	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	1900	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	1900	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	1900	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	1900	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	1900	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	1900	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	1900	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	1900	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	1900	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	16680	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	16680	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	16680	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	16680	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	16680	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	16680	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	16680	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	16680	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	16680	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	16680	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	16680	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	27461	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	41265	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	52111	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	52111	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	52111	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	52111	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:xxxx]	52111	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:xxxx]	52111	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:xxxx]	52111	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	59838	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	61017	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	61017	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	61017	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	61017	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:xxxx]	61017	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:xxxx]	61017	*	*											
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:xxxx]	61017	*	*											
BiglyBT.exe	28116	TCP	james-i7.abyss	30182	94.xxxxxxxx	35849	SYN_SENT										
BiglyBT.exe	28116	TCP	james-i7.abyss	30183	83.xxxxxxx	39349	SYN_SENT										
BiglyBT.exe	28116	TCP	james-i7.abyss	30184	162.xxxxxx	13536	SYN_SENT										
BiglyBT.exe	28116	TCP	james-i7.abyss	30185	42.xxxxxxx	19059	SYN_SENT										
BiglyBT.exe	28116	TCP	james-i7.abyss	30186	93.xxxxxxx	30572	SYN_SENT										
BiglyBT.exe	28116	TCP	james-i7.abyss	30187	172.xxxxxx	41265	SYN_SENT										
BiglyBT.exe	28116	TCP	james-i7.abyss	30188	172.xxxxxx	41265	SYN_SENT										
BiglyBT.exe	28116	UDPV6	[0:0:0:0:0:0:0:0]	49245	*	*											
BiglyBT.exe	28116	UDP	James-i7	49245	*	*											
BiglyBT.exe	28116	TCP	james-i7.abyss	30189	93.xxxxxxx	8999	SYN_SENT										
BiglyBT.exe	28116	TCP	james-i7.abyss	30190	136.xxxxxx	18039	SYN_SENT										


PROBLEM FOUND!

**Comment by parg** [comment]
Not sure you've found a problem, the only connections listed that are going through your crappy router are the few at the end that actually list (redacted) IP addresses. The others are just listening ports or loopback connectioins.

Some things to do to reduce network usage:

* Disable the 'Distributed Database (DHT) plugin
* Make sure you haven't installed the I2P Helper plugin
* Set max concurrent connection attempts to something low
* Set max global connections and per-torrent connections to something low

**Comment by crogonint** [comment]
No parg, those are the connections that hit outgoing IP addresses. There are also incoming connections, and a few oddball connection types. The program is still sending out 150 some connections trying to connect to every tracker on every service it can think of.

Important note: I DISABLED all of the extra services. BiglyBT turned them all back on!

Important note: I DISABLED all but one (1) tracker. Thankfully, BiglyBT didn't re-enable all of those.

Important note: That's all on one (1) torrent. Just imagine what two or three would do.

Importent note: As I mentioned above, my connections are already set ludicrously low. I'm not sure sure torrenting would even work at much fewer than 4/8 connections.

Important note: This isn't using a crappy router. It's using a software USB device. Think of it as a software Proxy Router trying to jam everything through a USB cable to another software devce. That's the choke point, not a router. Pretty sure it chokes out at around 30 connections. 

For the torrent I was using, I actually had to use DHT to re-connect to the only seeder I could connect with, continuously. Disabling DHT would have killed me completely.

I've never downloaded any add-ons that BiglyBT didn't offer to me. This laptop is new enough, I don't THINK it has any add-ons, but I wouldn't swear to it.

Stop assuming.

Bottom line, I can't use BiglyBT in anything resembling a sensible torrenting function as it stands. I can't start multiple torrents, I can't even seed multiple torrents. I can't do anything but attempt to get one single choked down torrent to finish.

Honestly, I don't know why I'm even trying here. I've been using BiglyBT since Vuse irritated me like 10 years ago. Maybe it's time to find a new client if you guys can't take customer issues seriously.

Seriously, the next time I need to download something over torrent, I'm not going to spend 3-4 days fighting BiglyBT. I'm going to find a new client. I'd bet money that the vast majority of people don't even bother telling you that they're having issues. They probably just uninstall and go find a better solution.

**Comment by parg** [comment]
Important note: Switch to a different BT client and stop wasting your time trying to use BiglyBT

**Comment by crogonint** [comment]
@parg #1600 
Thanks for answering my issues with well thought out responses and programatic changes to incorporate solutions to your design flaws.

Seriously. I'm out. Everyone is out. I'll never recommend this software to anyone ever again, and I'm going to advise all software sites to stop supporting it. ..or at least until someone responsible is in charge of the project.

Get a new job if you can't hack it as a software engineer. Design flaws don't stop existing because you click the close button, loser.
